README.md

```diff
@@ -168,6 +168,13 @@ This is a good example of an input series for testing a `range` query.
 * Rule files that can't be tested are listed in `test/conf/promtool_ignore`.
 * Rule files that can't be tested with a specific provider are listed in `test/conf/promtool_ignore_<provider>`.
 
+### Test inhibition labels
+
+One can check whether inhibition labels (mostly "cancel_if_" prefixed ones) are well defined and triggered by a corresponding label in the alerting rules by running the `make test-inhibitions` command at the projet's root directory.
+
+This command will output the list of missing labels. Each of them will need to be defined in the alerting rules. 
+If there is no labels outputed, this means all inhibition labels are defined.
```

Comment:
We should warn not to take its output too strictly, as it can be wrong for different reasons.

```suggestion
If there is no labels outputed, this means test did not find missing inhibition labels.

Warning: the tool may output false alerts or miss some alerts because of the following limitations.
- it does not check for rules that are only defined on some specific environments (like aws-specific rules)
- it tries to guess source labels rather than relying actual alertmanager inhibition, so may be wrong with some alerts  
```